StripeUICore/StripeUICore/Source/Elements/ElementsUI.swift

```diff
@@ -70,6 +70,37 @@ import UIKit
     public var cornerRadius = ElementsUI.defaultCornerRadius
     public var shadow: Shadow? = Shadow()
 
+    /// Checks if the theme is light.
+    ///
+    /// This uses the algorithm defined in https://www.w3.org/WAI/ER/WD-AERT/#color-contrast to
+    /// determine if the theme's background color is dark or light.
+    /// Defaults to `true` when it cannot be determined.
+    public func isLight(threshold: Float = 0.7) -> Bool {
```

Comment:
Could maybe reuse `STPColorUtils.swift`

Reply:
Didn't realize we already had an implementation of the algorithm. 
I do believe it makes sense to have method in the Theme to check if it is a dark or light theme though, so I'm gonna keep the method and just call the existing function in STPColorUtils.